src/types.ts

```diff
@@ -3,6 +3,8 @@ import { ExcalidrawElement } from "./element/types";
 export type AppState = {
   draggingElement: ExcalidrawElement | null;
   resizingElement: ExcalidrawElement | null;
+  // element being edited (e.g. text element)
```

Comment:
```suggestion
```

Reply:
We don't really need that comment.

Reply:
Right, I'll modify it but I'd still explain what it's for because it may not be obvious. E.g., does it apply when you're resising or not?